src/cider/tests/hashtable/CiderHashTableTest.cpp

```diff
@@ -68,8 +52,8 @@ TEST(CiderHashTableTest, factoryTest) {
   cider_hashtable::HashTableSelector<
       int,
       int,
-      MurmurHash,
-      Equal,
+      cider_hashtable::MurmurHash,
```

Comment:
`using namespace cider_hashtable`  instead?

Reply:
In  https://google.github.io/styleguide/cppguide.html#Namespaces says that "Do not use using-directives (e.g., using namespace foo)."